### PR TITLE
Predict outliers

### LINKED ISSUE #30: Predict outliers
We had clean the outliers to train the dataset. However, the test dataset had some outliers that our model can't predict very well. How can we deal with that?

### PR DESCRIPTION
**Pull request by Stefenon**
Fixes #30.

### CODE REVIEW

File: notebooks/outlier-treatment.ipynb [hunk 1]
```diff
@@ -0,0 +1,516 @@
+{
```
Comment (yvesfracari) [inline notebooks/outlier-treatment.ipynb:1]: <p>Can you change the notebook for our pattern?</p><p>the name could be: 1.3-vse-outliers-clean.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (yvesfracari) [inline notebooks/outlier-treatment.ipynb:1]: <p>Very clear description. You could insert the stats. score reference link like&nbsp;<a href="https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.zscore.html" target="_blank">this</a>.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (Stefenon) [reply notebooks/outlier-treatment.ipynb:1]: Sure, forgot about that.

Comment (Stefenon) [reply notebooks/outlier-treatment.ipynb:1]: Good idea!

**Review by yvesfracari — APPROVED** [review]

**Pull request merged**